Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    ColorSupplier colorSupplier;
+    public int getRandomNumber(){
+        Random random = new Random();
+        return random.nextInt(100);
+    }
+
+    public Ball getRandomBall(){
+        colorSupplier = new ColorSupplier();
+        Ball ball = new Ball();

```

Comment:
create constructor with these two fields and return directly - return new Ball(color, number);